libraries/mediapickers/api/src/main/kotlin/io/element/android/libraries/mediapickers/api/PickerLauncher.kt

```diff
@@ -25,11 +27,19 @@ class ComposePickerLauncher<Input, Output>(
     private val defaultRequest: Input,
 ) : PickerLauncher<Input, Output> {
     override fun launch() {
-        managedLauncher.launch(defaultRequest)
+        try {
+            managedLauncher.launch(defaultRequest)
+        } catch (activityNotFoundException: ActivityNotFoundException) {
+            Timber.w(activityNotFoundException, "No activity found")
```

Comment:
Should we use `toast` int these 2 cases too besides just logging the issue?

Reply:
The issue happen when the device has no Camera application, so I would not bother too much.